gitlabform/__init__.py

```diff
@@ -562,9 +562,9 @@ def _show_header(
         """
 
         if target == "ALL":
-            info(">>> Getting ALL groups and projects...")
+            info(">>> Getting ALL groups and projects that I can modify...")
```

Comment:
"Getting ALL groups and projects that I have permission to modify..."

Reply:
Thanks, corrected!